lib/std/ascii.zig

```diff
@@ -328,9 +328,10 @@ test "allocUpperString" {
 
 /// Compares strings `a` and `b` case-insensitively and returns whether they are equal.
 pub fn eqlIgnoreCase(a: []const u8, b: []const u8) bool {
+    if (a.ptr == b.ptr) return true;
```

Comment:
I think this should placed after the `len` check.
The slices may point at the same memory, but have different lengths.
Consider for example `eqlIgnoreCase("abc", "abc"[0..1])`

Reply:
Woah, never thought about that. Thanks for pointing out. Fixed now.